.github/build/action.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Build'
+inputs:
+  build_identifier:
+    description: 'If specified will be added to the version in order to identify non official releases'
+    required: false
+    default: ''
+outputs:
+  changelog_outputs_path:
+    description: 'path'
+    value: '${{ steps.changelog.outputs.path }}'
+  metadata_outputs_archive:
+    description: 'arhive'
+    value: '${{ steps.metadata.outputs.archive }}'
+  metadata_outputs_archive-checksum:
+    description: 'archive-checksum'
+    value: '${{ steps.metadata.outputs.archive-checksum }}'
+runs:
+  using: 'composite'
+  steps:
+    - name: Install dependencies
+      shell: bash
+      run: yarn install --frozen-lockfile
```

Comment:
Just out of interest: What is the difference between '--immutable' and '--forzen-lockfile'

Reply:
fine for me